DESCRIPTION

```diff
@@ -29,7 +29,7 @@ URL: https://github.com/insightsengineering/rtables,
     https://insightsengineering.github.io/rtables/
 BugReports: https://github.com/insightsengineering/rtables/issues
 Depends:
-    formatters (>= 0.5.5),
+    formatters (>= 0.5.5.9005),
```

Comment:
Usually we add regression tests with {formatters} PRs both in {rtables} and {rlistings}. They are usually where the issue was open first and where the bug is apparent. To avoid circular imports many tests in {formatters} are limited to fake and simplified tables/listings. To respond your question, I do not think there is really a need of dev version, but we (myself in this case) need to keep the versioning updated when updating tests in related PRs

Reply:
> To respond your question, I do not think there is really a need of dev version, but we (myself in this case) need to keep the versioning updated when updating tests in related PRs

I'm afraid that I don't follow. There is no need of dev version we but need to keep it updated? For what?

Just to be clear - those tests are run when using minimal version specification in DESC file - currently it is `0.5.5`. When using it - we fail. We must either alter test to be backwards compatible or increase the dependency.

Reply:
it is a regression test failing. Only solution is increase dependency

Reply:
thanks
that means that this PR is ready for review